### DISCUSSION TITLE
今日知ったエンジニア英語を共有し合うサービス

### DISCUSSION THREAD

**Discussion by rie-oki-agn** [commit-mate/commit-mate.net#22] [Projects]
Out of the box
platform agnostic 

みなさんはこの英語の意味わかりますか？！

英語のドキュメントを見るたび、新たに英語を知るので、せっかく知ったなら共有しあってみんなで知識を高め合うサービスあったらいいなと思いました！
略語とか全然わからない！
（社会貢献ではない）

**Comment by monsat** [comment]
とりあえずここでいろいろ投稿していこう！

**Comment by monsat** [comment]
**opinionated**

例) Vite is opinionated and comes with sensible defaults out of the box, but is also highly extensible via its [Plugin API](https://vitejs.dev/guide/api-plugin.html) and [JavaScript API](https://vitejs.dev/guide/api-javascript.html) with full typing support.

こだわりの強い
開発者の主張を大切にした
個性の強い

のような感じでしょうかねー

**Comment by sudo-roa** [comment]
I'll figure it out.
意味 : なんとかするよ。自分で解決策を見つけ出してみるというニュアンスらしいです。
プロダクトのアップデート勧告時の選択肢で目にしました！
サラッと使えたらかっこいいですよね！

**Comment by Pomerican** [comment]
**WET**
- DRY原則に背いたコード
- 同じ内容が繰り返されている時に、DRY原則と対比して言われる用語

使用例:
*WOW!君のコードはWETだね！！*

---
最近、ここが動いてなくて寂しかったので投稿してみました。
オシャレな表現多いですよね、プログラミング英語って🥰

**Comment by Pomerican** [comment]
コミットメッセージ:
｢fix some codes.｣
意味:
いくつかコードを修正したよ(ちょっとした崩れ、バグ解消含む)
本当は…:
何か不具合起きる(別箇所に影響、typo、リファクタリングしただけで直ってなくない？)

**Reply by monsat** [reply]
あー、「なんかおかしいから直したよ」っていう意味があるってことですかね？
